README.md

```diff
@@ -67,6 +65,17 @@ if the `@BUILTIN_TESTLISTS` keyword is used (in other words, when running tempes
 with a built-in test list file). Failure to do so will result in a wrong path
 error.
 
+### readonly-quick
+For quickly verifying the behavior of the objective cloud and making sure tempest
```

Comment:
Can we please keep the wording changes from the previous [PR](https://github.com/canonical/snap-tempest/pull/7#discussion_r1300536763)?
```suggestion
For quickly verifying the behavior of the target cloud and making sure tempest
```